cloudinit/net/dhcp.py

```diff
@@ -248,6 +248,8 @@ def dhcp_discovery(dhclient_cmd_path, interface, cleandir):
         else:
             ppid = util.get_proc_ppid(pid)
             if ppid == 1:
+                if len(pout) == 2:
+                    LOG.debug('dhclient error stream: %s', pout[1])
```

Comment:
Then this can simply be:

```suggestion
                LOG.debug('dhclient error stream: %s', stderr)
```